ztools/x509/json_test.go

```diff
@@ -33,6 +33,9 @@ func (s *JSONSuite) SetUpTest(c *C) {
 	}
 
 	for _, test := range tests {
+		if test.Name()[0] == 46 {
```

Comment:
What files are you looking for? I'd rather whitelist on suffix than blacklist on some random subset of prefixes.